bundles/best_practices.rst

```diff
@@ -166,6 +166,69 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+    env:
+        global:
+            - TEST_COMMAND="phpunit"
+
+    matrix:
+        fast_finish: true
+        include:
+              # Minimum supported PHP and Symfony version
+            - php: 5.5
+              env: DEPENDENCIES="minimum" COVERAGE=true TEST_COMMAND="phpunit --coverage-text"
+
+              # Test the latest stable release
+            - php: 5.5
+            - php: 5.6
+            - php: 7.0
+            - php: 7.1
+            - php: 7.2
+
+              # Test LTS versions
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:v2"
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:v3"
```

Comment:
Shouldn't also add `"symfony/lts": "^4@dev"` to ensure that it works when using only 4.x packages?

Reply:
normal jobs will already cover this, as 4.x will be the latest version anyway.

`"symfony/lts": "^4@dev"` will not forbid using a 3.x component anyway. It will only forbid using a 5.x one. The version 4 of the LTS package will be useless until we reach the 4.4 era (which is why it won't be stable until that time either)